qp_klp/tests/test_klp.py

```diff
@@ -395,6 +407,30 @@ def test_sequence_processing_pipeline(self):
 
         self.assertEqual(ainfo, exp)
 
+        # confirm that 'cmds.log' exists.
+        cmd_log_fp = join(self.out_dir, 'cmds.log')
+        self.assertTrue(exists(cmd_log_fp))
+
+        # confirm that fastq files were copied to uploads directory.
+        uploads_fp = ('/home/runner/work/qp-knight-lab-processing/'
```

Comment:
you need to take this path from the configuration, right? If not it's gonna fail in other systems.